### PR TITLE
mnad 22 nextjs 1

### PR DESCRIPTION
**Pull request by mnadolny8991**

### CODE REVIEW

**Comment by mnadolny8991** [comment]
@ababenko-solvd

File: homeworks/nextjs-1/src/styles/Gallery.module.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+.main {
+    margin-top: 80px;
+    margin-left: auto;
+    margin-right: auto;
+    width: fit-content;
+}
+
+.gallery {
+    width: 1360px;
```
Comment (ababenko-solvd) [inline homeworks/nextjs-1/src/styles/Gallery.module.css:9]: Why is this width here for?
It makes you application not responsive.

**Review by ababenko-solvd — APPROVED** [review]
Please share a link with the deployed app

File: homeworks/nextjs-1/src/styles/Gallery.module.css (hunk 1 above)
Comment (mnadolny8991) [reply homeworks/nextjs-1/src/styles/Gallery.module.css:9]: https://deployment-test-4fmc1hkwl-mnadolny8991s-projects.vercel.app/

**Pull request merged**